tests/simulation/physics/sources/test_temporal_profiles.py

```
@@ -0,0 +1,222 @@
+    sim_time = 12σ_t ensures the full pulse passes the detector before the
+    simulation ends (the pulse peaks at t0 = 6σ_t and is negligible after
+    t0 + 6σ_t).
+    """
+    objects, constraints, config, volume = _build_base()
+
+    _C = 3e8  # m/s
+    _SIGMA_F = 1e14  # Hz — spectral standard deviation
+    _F_CENTER = _C / _WAVELENGTH  # 3e14 Hz
+
+    sigma_t = 1.0 / (2 * np.pi * _SIGMA_F)  # ~1.59 ps
+    sim_time = 12 * sigma_t  # t0=6σ_t peak + 6σ_t tail ≈ 19 ps
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)